jargons/argument.md

```diff
@@ -0,0 +1,25 @@
+---
+title: argument
+sameWith:
+- parameter
+- parametre
+tags:
+- genel
+---
+
+Argument, bir fonksiyona gönderilebilen ve fonksiyon tarafından işlenebilen değerdir. ``parameter`` ya da ``parametre`` olarak da kullanılır. 
+
+```php
```

Comment:
Site genelinde örnekleri javascript yazmak daha faydali olacak diye dusunuyorum. Tabi yazilan jargon sadece belli bir dile ait veya JavaScript'te karsiligi yoksa baska bir dil kullanilabilir. Emin olmamakla birlikte bir de kod içinde fonksiyon, degisken tanimlamalari ingilizce yapilirsa insanlar baska kaynaklar okurken burada gorduklerini daha rahat animsarlar diye tahmin ediyorum.

Son olarak burada array'ler foreach'ler ile kafa karistirmaya gerek yok diye dusunuyorum.

```js
// x, add fonksiyonun ilk, y ise ikinci argument'idir.
const add = (x, y) => x + y;
```

Soyle bir ornek yeterli olacaktir. Buradan konu ile alakasi olmayan array'a referans vermek gereksiz bana kalirsa.

Reply:
Ben javascript ile yazilmasi gerektigine katilmiyorum. Genel olarak herkesin anlayabilecegi birkac dilden biri oldugu surece sorun olmaz sonucta. Birileri gelip farkli dillerden ornekler ekleyerek de gelistirebilir.

Reply:
Tabi iki veya daha fazla dilden ornek olmasi en guzeli ama sadece tek bir kod ornegi varsa onun da js olmasi daha iyi olur diye dusunmustum ben herkesin az cok syntax'ini bildigi icin.

Degisken, fonksiyon isimlendirmelerinin Ingilizce olmasi hakkinda ne dusunuyorsun bu arada? Ben sanki insanlar baska kaynaktan tutorial okurken Ingilizce gorecekleri kullanimlari burada da Ingilizce gorseler daha rahat anlarlar diye dusunuyorum.

Reply:
Temel olarak PHP kod yazdığım için örneği öyle çıktım ama javascript için olan örnek ile değiştiririm. Fonksiyon isimleri ve diğerleri için de notumu aldım 👍 düzenliyorum